rest/resources/rest.json

```diff
@@ -1,7 +1,8 @@
 {
   "network": {
     "name": "mijinTest",
-    "description": "catapult development network"
+    "description": "catapult development network",
+    "maxRollBackBlocks": 300
```

Comment:
Since the node operator might change this value might be changed and not correspond to the network ``maxRollBackBlocks``, I would suggest moving this property outside ``network`` and call it ``transactionFeesBlocks``.

Reply:
Just wonder, it should use `maxDifficultyBlocks ` rather than `maxRollBackBlocks` ?

Reply:
> Since the node operator might change this value might be changed and not correspond to the network `maxRollBackBlocks`, I would suggest moving this property outside `network` and call it `transactionFeesBlocks`.

thought i answered this yesterday but i just see that it was not really posted -_- i agree with moving it outside of network, though i'd try to come up with something different from both of your suggestions that expresses what this seating really does, definitely not `maxRollBackBlocks` because it can be confused with the server setting, `transactionFeesBlocks` sounds better but i'd still add some extra words to make it more self-explanatory, even if a bit long of a setting name

Reply:
let's use `numBlocksForTransactionFeeStats` and close this